### ISSUE TITLE
Programming language agnostic

### ISSUE THREAD

**Issue by MarcOedingen** [MarcOedingen/ChatGPT-Code-Detection#8]
We are currently training our model exclusively on Python code samples. In an earlier test, however, we have already established that this methodology is transferable to Java, so that we were able to achieve similar accuracy to that already documented in the paper even without fine-tuning. It is therefore quite conceivable that we will also be able to work with other programming languages.
